examples/lock-app/nxp/mcxw71/include/config/CHIPProjectConfig.h

```diff
@@ -172,12 +172,12 @@
 #define CHIP_DEVICE_CONFIG_BLE_ADVERTISING_TIMEOUT (15 * 60 * 1000)
 
 /**
- * CONFIG_CHIP_NFC_COMMISSIONING, CHIP_DEVICE_CONFIG_ENABLE_NFC
+ * CONFIG_CHIP_NFC_ONBOARDING_PAYLOAD, CHIP_DEVICE_CONFIG_ENABLE_NFC_ONBOARDING_PAYLOAD
  *
- * NFC commissioning is not supported on K32W1
+ * NFC onboarding is not supported on K32W1
```

Comment:
```suggestion
 * NFC onboarding payload is not supported on K32W1
```
This is getting a bit repetitive....